src/current/_includes/v24.1/physical-replication/fast-cutback-syntax.md

```diff
@@ -0,0 +1,8 @@
+{% include_cached new-in.html version="v24.1" %} To cut back to a cluster that was previously the primary cluster, use the [`ALTER VIRTUAL CLUSTER`]({% link {{ page.version.version }}/alter-virtual-cluster.md %}) syntax:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+ALTER VIRTUAL CLUSTER original_primary_vc START REPLICATION FROM promoted_standby_vc ON connection_string_standby;
```

Comment:
Should the placeholders be surrounded by curly braces?

Reply:
Sure!